frontend/src/components/editor/EditorBottomBar.tsx

```diff
@@ -0,0 +1,68 @@
+import { Button, Menu, MenuItem } from "@mui/material";
+import { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { CodeKeyType, selectEditor, setCodeKeyType } from "../../store/editorSlice";
+
+export const BOTTOM_BAR_HEIGHT = 25;
+
+const EditorBottomBar = (props: { width: number | string }) => {
```

Comment:
Please change it with `function` keyword.
Also, we define the props type like below code.

```typescript
interface EditorBottomBarProps {
    width: number | string
}
```